cli/constants/constants.go

```diff
@@ -38,17 +39,19 @@ var (
 				"chopsticks":  3000,
 			},
 			"liquid": map[string]int{
-				"peer":        7040,
-				"node":        7041,
-				"esplora":     5001,
-				"electrs":     3012,
-				"electrs_rpc": 60401,
-				"chopsticks":  3001,
+				"peer":         7040,
+				"node":         7041,
+				"esplora":      5001,
+				"electrs":      3012,
+				"electrs_rpc":  60401,
+				"chopsticks":   3001,
+				"liquid-coach": 7000,
```

Comment:
maintain the style that you find already in the codebase ie. underscore instead of dash